packages/nextjs/utils/impactCalculator/common.ts

```diff
@@ -0,0 +1,7 @@
+export const abbreviateNumber = (num: number) => {
```

Comment:
Considering your comment about it seeming unconventional to have 3000k can we make this roll over to 1m when we get past 999k? Maybe use decimals for numbers between, like 1.25m, 1.6m etc. 

I like your approach with using abbreviations to make space for the label.

Reply:
Yeah great call